dbtype/mssql/db_report_string.sh

```diff
@@ -7,9 +7,9 @@ db_report_string() {
   then
     ${db_binary} -d ${deployment_db} ${server_flag}${port_flag} ${user_flag} ${password_flag} -h -1 -b -Q "
     SET NOCOUNT ON;
-    SELECT '${db_basedir}/' + dbname + '/' + deployment_type + '/' + deployment_name
+    SELECT '${db_basedir}/' + ${dbname} + '/' + deployment_type + '/' + deployment_name
```

Comment:
ok, forget everything I said there.  You're doing the right thing, but the string you're passing in needs wrapped in single quotes in the query.  Previously it didn't need them because it was a column.  You have it correct in the PG version.